### PR TITLE
[Merged by Bors] - Automatically assign backlog tasks

### PR DESCRIPTION
**Pull request by fpoli**
With this PR, a github action will open a PR every 2 weeks asking one of us to fix a panic/error in the test-crates job. Any comments?

### CODE REVIEW

**Comment by vakaras** [comment]
I would prefer to have such a CI task after refactoring is finished.

File: .github/workflows/backlog.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          # Use viper-admin's token to workaround a restriction of GitHub.
+          # See: https://github.com/peter-evans/create-pull-request/issues/48
+          token: ${{ secrets.VIPER_ADMIN_TOKEN }}
+          commit-message: Backlog fixes
+          title: Backlog fixes
+          branch: auto-backlog-${{ env.DATE }}
+          delete-branch: true
+          assignees: ${{ env.DEVELOPER }}
+          body: |
+            1. In `test-crates/crates.csv`, replace one `Skip` with `NoCrash` or one `NoCrash` with `NoErrors`.
```
Comment (fpoli) [inline .github/workflows/backlog.yml]:
```suggestion
            1. In `test-crates/crates.csv`, replace one `Skip` with `NoCrash` or one `NoCrash` with `NoErrors`. Optionally, also update the version of the crate.
```

File: .github/workflows/backlog.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          commit-message: Backlog fixes
+          title: Backlog fixes
+          branch: auto-backlog-${{ env.DATE }}
+          delete-branch: true
+          assignees: ${{ env.DEVELOPER }}
+          body: |
+            1. In `test-crates/crates.csv`, replace one `Skip` with `NoCrash` or one `NoCrash` with `NoErrors`. Optionally, also update the version of the crate.
+            2. Test the corresponding crate (`./target/release/test-crates <name_of_the_crate>`).
+            3. If the test succeeds, go back to step 1 and repeat.
+            4. If the test fails, fix at least one bug (e.g. convert a panic to an internal error or an internal error to a readable "unsupported" error message).
```
Comment (fpoli) [inline .github/workflows/backlog.yml:38]:
```suggestion
            4. If the test fails, fix at least one bug (e.g. convert a panic to an internal error or an internal error to a readable "unsupported" error message).
            5. At this point, if your fix alone is not enough to make all the tests pass, it's fine to revert your latest change of step 1.
```

File: .github/workflows/backlog.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name: Backlog fixes
+
+on:
+  workflow_dispatch:
+  schedule:
+    # At 07:00 UTC on day-of-month 1 and 15. Use https://crontab.guru/ to edit this.
+    - cron:  '0 7 1,15 * *'
+
+jobs:
+  # Update the version of rustc, open a PR and assign a developer.
```
Comment (fpoli) [inline .github/workflows/backlog.yml]:
```suggestion
```

**Pull request closed without merging**